src/Polly.Core/Retry/RetryHelper.cs

```diff
@@ -25,7 +25,53 @@ public static TimeSpan GetRetryDelay(RetryBackoffType type, int attempt, TimeSpa
             RetryBackoffType.Linear => (attempt + 1) * baseDelay,
             RetryBackoffType.Exponential => Math.Pow(ExponentialFactor, attempt) * baseDelay,
 #endif
+            RetryBackoffType.ExponentialWithJitter => DecorrelatedJitterBackoffV2(attempt, baseDelay, ref state, random),
             _ => throw new ArgumentOutOfRangeException(nameof(type), type, "The retry backoff type is not supported.")
         };
     }
+
+    /// <summary>
+    /// Generates sleep durations in an exponentially backing-off, jittered manner, making sure to mitigate any correlations.
+    /// For example: 850ms, 1455ms, 3060ms.
+    /// Per discussion in Polly issue 530, the jitter of this implementation exhibits fewer spikes and a smoother distribution than the AWS jitter formula.
```

Comment:
Lets make "Polly issue 530" a link to the actual issue?

Reply:
Done.